Platform/ARM/VExpressPkg/Library/ArmVExpressPciHostBridgeLib/ArmVExpressPciHostBridgeLib.c

```diff
@@ -95,11 +95,9 @@ STATIC PCI_ROOT_BRIDGE mRootBridge = {
   (EFI_DEVICE_PATH_PROTOCOL *)&mEfiPciRootBridgeDevicePath
 };
 
-#ifndef MDEPKG_NDEBUG
 STATIC CONST CHAR16 mAcpiAddrSpaceTypeStr[][4] = {

```

Comment:
Is this change same as https://github.com/tianocore/edk2-platforms/pull/230/commits/66977b25fdc18bc00054eb775516744dbe85b8d4#diff-e5328c52eaaf9eb7b837df68d835a204849ea1cc62975a697b29de7829e08042 ?

Reply:
Ah. Yes. If this patch is merged. the problem will be gone :)